aten/src/ATen/native/quantized/README.md

```diff
@@ -188,9 +188,13 @@ namespace at {
     return {std::forward<Inputs>(inputs)...};
   }
 
-  /* Given an operator handle, calls it using some arguments.*/
+  /* Given an operator handle, calls it using some arguments.
+     Note: this is slow because it boxes every argument
+     and likely has to re-unbox them when calling the kernel.
+     Prefer calling c10::OperatorHandle::callUnboxed<Args...>(args...).
+  */
   template <class... Args>
-  inline std::vector<torch::IValue> callOp(
+  inline std::vector<torch::IValue> callOp_slow(
```

Comment:
These are docs... can't you just update the docs so that they do things the right way?

Reply:
ok I reverted the changes here for now because I'm only 99% (but not 100%) confident that op.callUnboxed works for all use cases where callOp worked.